Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.10.8-slim
+LABEL authors="zazmargo@gmail.com"
+
+ENV PYTHOUNNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt

```

Comment:
Ensure that the `requirements.txt` file exists in the context of the Docker build. If it doesn't, the build will fail at this step.